src/app/pages/client/suggestions/suggestions.page.ts

```diff
@@ -0,0 +1,91 @@
+import { Component, OnInit } from '@angular/core';
+import {Router} from "@angular/router";
+import {NotificationService} from "../../../services/notification.service";
+import {SuggestionService} from "../../../services/suggestion.service";
+
+@Component({
+  selector: 'app-suggestions',
+  templateUrl: './suggestions.page.html',
+  styleUrls: ['./suggestions.page.scss'],
+})
+export class SuggestionsPage implements OnInit {
+  availableSuggestions : any[] = [];
+  votedSuggestions     : any[] = [];
+
+  constructor(
+      private suggestions : SuggestionService,
+      private router      : Router,
+      private notification: NotificationService
+  ) {}
+
+  /**
+   * Updates the list of question
+   */
+  private async updateSuggestions(){
```

Comment:
Same comment here  about `this.availableSuggestions` array being empty on `this.suggestions.getSuggestions()` error returning an empty array.

Reply:
I think the result of `this.questions.getQuestions()` should be trusted as it comes from the server. If the server replies without sending any questions, we should assume there are no questions for a valid reason.

Updated the doc in c275629.